browser/skus/sources.gni

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2022 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+brave_browser_skus_sources = [
+  "//brave/browser/skus/skus_service_factory.cc",
```

Comment:
Moved skus factory to source.gni to use g_browser_process->local_state() from #include "chrome/browser/browser_process.h"